itou/siaes/management/commands/_import_siae/convention.py

```diff
@@ -76,37 +66,27 @@ def update_existing_conventions():
                 pass
             else:
                 # Active convention should be deactivated.
-                deactivations += 1
-                # Break if too many deactivations have occurred without waiting for the end of the loop.
-                # This way we avoid shutting down 100% of our conventions on January 1st of the year.
-                assert deactivations <= 200
-                deactivations_by_kind[convention.kind] += 1
-                if DEACTIVATE_CONVENTIONS:
-                    convention.is_active = False
-                    # Start the grace period now.
-                    convention.deactivated_at = timezone.now()
-                    convention.save()
+                conventions_to_deactivate.append(convention)
 
     print(f"{reactivations} conventions have been reactivated")
 
-    total = SiaeConvention.objects.count()
-
-    if DEACTIVATE_CONVENTIONS:
-        print(f"{deactivations} of {total} conventions have been deactivated")
+    if len(conventions_to_deactivate) >= 200:
+        # Early each year, all or most AF for the new year are missing in ASP AF data.
+        # Instead of brutally deactivating all SIAE, we patiently wait until enough AF data is present.
+        # While we wait, no SIAE is deactivated whatsoever.
+        print(
+            f"ERROR: too many conventions would be deactivated ({len(conventions_to_deactivate)})"
+            f" thus none will actually be!"
+        )
         return
 
-    print(f"{deactivations} of {total} conventions should have been deactivated but have *not* been")
+    for convention in conventions_to_deactivate:
+        convention.is_active = False
+        # Start the grace period now.
+        convention.deactivated_at = timezone.now()
+    SiaeConvention.objects.bulk_update(conventions_to_deactivate, ["is_active", "deactivated_at"])
```

Comment:
J'ai eu des soucis en ne mettant pas le batch_size pour éviter le timeout error
```suggestion
    SiaeConvention.objects.bulk_update(conventions_to_deactivate, ["is_active", "deactivated_at"], batch_size=200)
```